public/lang/babele/de/investigator.pathOfCthulhuAbilities.json

```diff
@@ -178,14 +178,14 @@
         },
         "Oral History": {
             "name": "Zeugenbefragung",
-            "category": "Sozial"
+            "category": "Zwischenmenschlich"
         },
         "Outdoorsman": {
-            "name": "Naturkunde",
+            "name": "Naturbursche",
             "category": "Technisch"
         },
         "Pharmacy": {
-            "name": "Pharmazie",
+            "name": "Arzneimittelkunde",
```

Comment:
Again:Lets stay with common terms. Pharmazie is already established in CoC. Also Arzneimittelkunde is pretty long wich may lead to space issues on sheets